src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityCreativeTank.java

```diff
@@ -71,22 +69,22 @@ protected ModularUI createUI(EntityPlayer entityPlayer) {
                 () -> this.fluidTank.getFluid(), data -> {
             this.fluidTank.setFluid(data);
         })
-                .showTip(false).setBackgroundTexture(GuiTextures.SLOT_DARKENED));
+                .showTip(false));
```

Comment:
The formatting here looks rather off, in my opinion; it should attach to the close parenthesis of the previous line.